lib/fixup.rb

```diff
@@ -128,14 +128,7 @@
   print "\nWould you like to remove deprecated package #{pkg[:pkg_name].capitalize}? [y/N] "
   case $stdin.gets.chomp.downcase
   when 'y', 'yes'
-    @in_fixup = true
-    begin
-      remove pkg[:pkg_name]
-    rescue NoMethodError # This won't work the first time crew update happens, since this requires an update to crew.
-      puts "Please rerun 'crew update' to allow for removal of #{pkg[:pkg_name].capitalize}.".orange
-      puts "#{pkg[:pkg_name].capitalize} not removed.".lightblue
-    end
-    @in_fixup = false
+    system("crew remove #{pkg[:pkg_name]}")
```

Comment:
Didn't we have to refactor due to environment issues in the containers because we ran crew within crew @satmandu?

Reply:
We do a similar thing at line 1665 of `bin/crew`:
```ruby
system 'yes | crew install ruby_rubocop' unless @device[:installed_packages].any? { |elem| elem[:name] == 'ruby_rubocop' }
```

Though I'm not aware of these issues so not sure if this would trigger it.

Reply:
Yes that was a problem! We should fix line 1665! invoking crew within crew needs to be fixed, but currently it breaks things...

Reply:
Admittedly I don't think line 1665 ever gets called, as rubocop is in buildessential.

Reply:
@satmandu: What is your recommendation?  This is the only thing I could find in this PR where there might be a potential problem.

Reply:
Fixed.